### PR TITLE
{data}[GCCcore/8.2.0] BeautifulSoup v4.8.0 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/c4b53aede0bb1b2310048f8f53ba0087 for a full test report.

File: easybuild/easyconfigs/b/BeautifulSoup/BeautifulSoup-4.8.0-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+version = '4.8.0'
+
+homepage = 'https://www.crummy.com/software/BeautifulSoup'
+description = "Beautiful Soup is a Python library designed for quick turnaround projects like screen-scraping."
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+builddependencies = [('binutils', '2.31.1')]
+
+dependencies = [('Python', '3.7.2')]
```
Comment (Micket) [inline easybuild/easyconfigs/b/BeautifulSoup/BeautifulSoup-4.8.0-GCCcore-8.2.0.eb]: Seems like it would support multi_deps

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3116.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/e39084824f4652169ce786c226ed3267 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2065.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1bfd392e664d9f615c6626f874dd0d5b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/a723f772a25fbb4c6f407d902c021a38 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/fac75c6a6b9dcfd9a97334f050b28f1f for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/2abfa76a0e58f57f1f625c750bcbbb04 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**